libraries/AP_NavEKF2/AP_NavEKF2_core.cpp

```diff
@@ -762,7 +762,7 @@ void NavEKF2_core::calcOutputStates()
     // Coefficients selected to place all three filter poles at omega
     const float CompFiltOmega = M_2PI * constrain_float(frontend->_hrt_filt_freq, 0.1f, 30.0f);
     float omega2 = CompFiltOmega * CompFiltOmega;
-    float pos_err = outputDataNew.position.z - vertCompFiltState.pos;
+    float pos_err = constrain_float(outputDataNew.position.z - vertCompFiltState.pos, -1e5, 1e5);
```

Comment:
Why not float_espilon ?

Reply:
it's 1e5 not 1e-5 (ie. we're constraining not to be too big, not to prevent it being too small)
1e5 is 10km of height error

Reply:
Ohh right... My bad